pylint/lint/parallel.py

```diff
@@ -62,9 +63,13 @@ def _worker_initialize(linter, arguments=None):
     _patch_sys_path(arguments or ())
 
 
-def _worker_check_single_file(file_item):
+def _worker_check_single_file(
+    file_item: Tuple[str, str, str]
```

Comment:
Does it make sense to define the type for `file_item` as NamedTuple? It seems to be used in a few places.

Reply:
Good call!
Probably after https://github.com/PyCQA/pylint/pull/4973 though to add it to the `typing.py` document.